docs/src/legacy/child_device_config_management_agent.md

```diff
@@ -72,7 +72,7 @@ to the URL: `http://{fts-address}:8000/tedge/file-transfer/{child-id}/c8y-config
 * {fts-address}` is the address of the thin-edge.io device on which the [File Transfer Service][fts] is running
 * `{child-id}` is the child-device-id
 
-[fts]: ../../references/tedge-file-transfer-service.md
+[fts]: ../references/tedge-file-transfer-service.md
```

Comment:
I think we can delete this line as it is invalid markdown (to my knowledge) and it is not rendered on the page anyway...

Reply:
This is a so called [reference link](https://daringfireball.net/projects/markdown/syntax#link).

As this is unusual, I removed it.